ramls/group-fund-fiscal-year.raml

```diff
@@ -0,0 +1,75 @@
+#%RAML 1.0
+
+title: Finance group fund fiscal year
+version: v1
+protocols: [ HTTP, HTTPS ]
+baseUri: https://github.com/folio-org/mod-finance
+
+documentation:
+  - title: Group Fund Fiscal Year API
+    content: This documents the API calls that can be made to manage group-fund-fiscal-years
+
+types:
+  groupFundFiscalYear: !include acq-models/mod-finance/schemas/group_fund_fiscal_year.json
+  groupFundFiscalYearCollection: !include acq-models/mod-finance/schemas/group_fund_fiscal_year_collection.json
+  errors: !include raml-util/schemas/errors.schema
```

Comment:
Don't think `errors` is not used?

Reply:
`errors` is used in the validation trait https://github.com/folio-org/raml/blob/a131bc9787dfcda9cf093d200a2fc20cd295512a/traits/validation.raml#L7